stable/ipfs/templates/_helpers.tpl

```diff
@@ -0,0 +1,31 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a name for the service to use.
+
+We allow overrides for name of service, since this service provides an API that could be directly
+called by end-users. For example, you could call it 'ipfs' and then all users running in a namespace
+could just connect to it by specifying 'ipfs'.
+*/}}
+{{- define "servicename" -}}
+{{- if .Values.service.nameOverride -}}
```

Comment:
On second thought, am not sure this needs truncation. The user is responsible for setting it, and the default fullname handles truncation already.